packages/http-client-csharp/generator/Packages.Data.props

```diff
@@ -9,12 +9,12 @@
     <PackageReference Update="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.4" />
     <PackageReference Update="CommandLineParser" Version="2.9.1" />
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.14.0" />
-    <PackageReference Update="Microsoft.Build" Version="17.9.5" />
+    <PackageReference Update="Microsoft.Build" Version="17.14.8" />
     <PackageReference Update="NuGet.Configuration" Version="6.14.0" />
     <PackageReference Update="NuGet.Versioning" Version="6.14.0" />
     <PackageReference Update="NuGet.Protocol" Version="6.14.0" />
-    <PackageReference Update="System.ComponentModel.Composition" Version="8.0.0" />
-    <PackageReference Update="System.ClientModel" Version="1.4.1" />
-    <PackageReference Update="System.Memory.Data" Version="8.0.1" />
+    <PackageReference Update="System.ComponentModel.Composition" Version="9.0.6" />
+    <PackageReference Update="System.ClientModel" Version="1.4.2" />
+    <PackageReference Update="System.Memory.Data" Version="9.0.6" />
```

Comment:
I'm assuming these are the dependencies consumed by the generator, not injected by.